package.json

```diff
@@ -64,7 +64,6 @@
     "highlight.js": "^10.4.1",
     "hoek": "~4.2.0",
     "elliptic": "~6.5.4",
-    "react-dev-utils": "~11.0.4",
```

Comment:
This was un-needed as it already was correctly resolving to `11.0.4`in the lock file and it was interfering with resolutions further downstream for `minimatch`.